deployment/v3/mosip/partner-onboarder/install.sh

```
@@ -70,7 +70,7 @@ function installing_onboarder() {
     done
 
   NS=onboarder
-  CHART_VERSION=1.3.0-beta.1
+  CHART_VERSION=1.3.0-develop
```

Shellcheck (0.11.0)

[warning] CHART_VERSION appears unused. Verify use (or export if used externally).

(SC2034)